sw/device/silicon_creator/lib/drivers/pinmux.c

```diff
@@ -114,6 +115,61 @@ static void enable_pull_down(top_earlgrey_muxed_pads_t pad) {
       kBase + PINMUX_MIO_PAD_ATTR_0_REG_OFFSET + pad * sizeof(uint32_t), reg);
 }
 
+/**
+ * Enables or disables pull-up/pull-down fro the specified pad.
+ *
+ * @param pad A MIO pad.
+ * @param enable Whether the internal pull resistor should be enabled.
+ * @param up Whether the pull resistor should pull up(true) or down(false).
+ */
+static void enable_pull(top_earlgrey_muxed_pads_t pad, bool enable, bool up) {
```

Comment:
This function seems to subsume `enable_pull_down`. Should the old one be removed and its usage converted?